### PR TITLE
updated intelpython3_full dockerfile

### PR DESCRIPTION
**Pull request by devpramod-intel**
The current intelpython3_dockerfile is outdated. The updated dockerfile pulls the latest image from miniconda and installs the latest available version of intelpython3_full

### CODE REVIEW

**Comment by xaleryb** [comment]
@aguzmanballen  could you please have a look if this change is good enough or there is something more we need to do?

**Review by AndresGuzman-Ballen — CHANGES_REQUESTED** [review]
Thanks for creating the PR! Would you be able to take a look at the one comment I made?

File: configs/intelpython3_full/Dockerfile [hunk 1]
```diff
@@ -23,13 +23,9 @@
...
+FROM continuumio/miniconda3
 
-# This file is automatically generated from tpl.Dockerfile
-
-ENV ACCEPT_INTEL_PYTHON_EULA=yes
 
 RUN conda config --add channels intel
-RUN conda install  -y -q intelpython3_full=2021.4 python=3
-RUN conda clean --all
+RUN conda install  -y -q intelpython3_full
```
[the remark this answers is not in the document]
Comment (devpramod-intel) [reply configs/intelpython3_full/Dockerfile]: @AndresGuzman-Ballen Thank you! I have made the change

[the remark this answers is not in the document]
Comment (AndresGuzman-Ballen) [reply configs/intelpython3_full/Dockerfile]: @devpramod-intel thank you!

**Pull request merged**